cmd/osbuild-worker/main.go

```diff
@@ -430,13 +435,17 @@ func main() {
 				CABundle:            genericS3CABundle,
 				SkipSSLVerification: genericS3SkipSSLVerification,
 			},
+			ContainerAuthFile: containersAuthFilePath,
 		},
 		worker.JobTypeKojiInit: &KojiInitJobImpl{
 			KojiServers: kojiServers,
 		},
 		worker.JobTypeKojiFinalize: &KojiFinalizeJobImpl{
 			KojiServers: kojiServers,
 		},
+		worker.JobTypeContainerResolve: &ContainerResolveJobImpl{
+			AuthFilePath: containersAuthFilePath,
+		},
 	}
```

Comment:
For future reference: I think we should not have this be depsolve v non-depsolve jobs, but rather build v non-build jobs. I.e., put all the short-running ones together as their latency will be much shorter.

Reply:
Makes sense